elements/pfe-sass/extends/_typography-deprecated.scss

```diff
@@ -0,0 +1,39 @@
+// DEPRECATED 
+%pfe-headline-xxl  {
+    font-family: pfe-var(font-family--heading);
+    font-size:   pfe-var(font-size--xxl);  
```

Comment:
note that these deprecated classes extend plain "font-size" vars as opposed to "font-size--heading" vars

Reply:
Can you tell me more about why we need to add deprecated extends?  I understand the deprecated classes but why add extends that are already deprecated?

Reply:
fair point, I can just put the value of the extends here